### ISSUE TITLE
API: fix CORS

### ISSUE THREAD

**Issue by aguillon** [marigold-dev/gas-station#3]
Right now allowed origins are hardcoded. What should we do instead? Should we just allow *?

**Comment by aguillon** [comment]
@quentin-burg do you think this one is done?

**Comment by quentin-burg** [comment]
@aguillon I don't think so because for the moment we just allow all origins to test and develop the MVP but maybe there is something better to do than that.
I think it's not urgent but we will have to look at this issue

**Comment by ulrikstrid** [comment]
Might be a good idea to allow setting URLs when registering a contract and add those to cors. But might be overkill?

**Comment by aguillon** [comment]
@ulrikstrid I'm not sure I see why it matters who's targeting the API. Do you mean that it would allow clients (= users who sponsor operations) to make sure only the users of their website are being paid for?

**Comment by ulrikstrid** [comment]
We can disallow "custom clients" for the contract. But again, it might be overkill.

**Comment by aguillon** [comment]
I've asked a friend who's into webdev and he says we can allow *.
